woocommerce/woocommerce-gateway-stripe
# Display Apple Pay button with text based on branded_type

Fixes #1217.

#### Changes proposed in this Pull Request:

- Use `branded_type` to decide which variation of the payment button is going to be rendered

#### Testing instructions:

1. Have "Branded" set as the "Payment Request Button Type"
2. Have "Text and logo" set as the "Payment Request Branded Button Label Format"
3. Check a product page using Apple Pay
4. Verify that the button includes the "Buy with" text
